FirebaseAuth/Sources/Swift/AuthProvider/OAuthProvider.swift

```diff
@@ -78,6 +78,7 @@ import Foundation
     return OAuthProvider(providerID: providerID.rawValue, auth: auth)
   }
 
+  /// Initializes an `OAuthProvider`.
```

Comment:
I can't deprecate this API easily because doing so would mean I need to delegate to another initializer which would make this initializer a `convenience` initializer, which could break subclasses (example in our testing infra).